packages/instui-cli/lib/commands/create.js

```diff
@@ -63,35 +64,50 @@ exports.builder = (yargs) => {
   )
 
   const packageCommand = 'package'
+  const packageFormatInstructions = ' (in kebab-case, e.g. my-package)'
   yargs.command(
     packageCommand,
-    'Create an Instructure UI package. DEPRECATED, this will be deleted in InstUI 9',
+    'Create an Instructure UI package. FOR INTERNAL USE ONLY!',
```

Comment:
I am not sure if we truly want to have this internally only. I think anyone could and should use this, just don't assume that it won't have breaking changes